bootloader/hos/hos.c

```diff
@@ -638,7 +645,7 @@ int hos_launch(ini_sec_t *cfg)
 	_free_launch_components(&ctxt);
 
 	// Copy BCT if debug mode is enabled.
-	memset((void *)0x4003D000, 0, 0x3000);
+	// memset((void *)0x4003D000, 0, 0x3000);
```

Comment:
This needs to be addressed. The original patch usage is not OK.
check if != vers 6. If it is, memset.